src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs

```diff
@@ -4998,10 +4998,7 @@ class Derived : Base<int>
             var compilation = CreateCompilation(text);
             if (compilation.Assembly.RuntimeSupportsCovariantReturnsOfClasses)
             {
-                // We no longer report a runtime ambiguous override because the compiler
-                // produces a methodimpl record to disambiguate.
-                compilation.VerifyDiagnostics(
-                    );
+                throw new Exception("Is this reachable?");
```

Comment:
Since this is a test, I was wondering if you could just run the test locally?

Reply:
@alrz Already done, and... it passes :(

![image](https://user-images.githubusercontent.com/31348972/118232304-9ef3cc00-b490-11eb-8036-33d82a347d07.png)